code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -156,20 +156,14 @@ Contains most of the procs that are called when a mob is attacked by something
 
 /mob/living/carbon/human/emp_act(severity)
 	. = ..()
-	for(var/obj/O in src)
-		if(!O)
-			continue
-		O.emp_act(severity)
 	for(var/obj/limb/O in limbs)
 		if(O.status & LIMB_DESTROYED)
 			continue
-		O.emp_act(severity)
 		for(var/datum/internal_organ/I in O.internal_organs)
 			if(I.robotic == FALSE)
 				continue
 			I.emp_act(severity)
```

Comment:
Move this to
`/obj/limb/emp_act(severity)`
and then the `/mob/living/carbon/human/emp_act(severity)` can be deleted entirely.